runtime/queries/go/highlights.scm

```diff
@@ -157,3 +190,7 @@
 ] @constant.builtin
 
 (comment) @comment
+
+; Error
+
+(ERROR) @error
```

Comment:
For this PR and #6203: I'm not sure we should add the `(ERROR) @error` scopes. We've discussed this before and it might be best to remove them from all queries and add an option that automatically adds the highlight: https://github.com/helix-editor/helix/pull/3789#discussion_r967739912

Reply:
The ERROR section has been removed.